README.md

```diff
@@ -1 +1,43 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Introduction
+
+My name is Vanessa Guigon and I am from *Chicago, Illinois*. I
+am a rising third year at UChicago majoring in **Public Policy** 
+and **Political Science**. 
+
+### About Me
+
+#### Fun Facts:
+- I was born in Florida and I moved to Chicago when I was young.
+- I am an identical twin.
+  - I am older than her by *16 minutes*.
+- I am a Dual Citizen of the **US** and **France**.
+
+#### Photo:
+Below is an image of myself.
+
+![](Me.jpg)
```

Comment:
This image doesn't come up when the page is viewed. I think you might have used a picture that is located on your computer, rather than uploaded to your repository. I also ran into this problem at first and looked at the 'issues' section of the cfss-su20 version of HW01 on how to push images from the computer to the repository. From there, the link from the homework assignment on the class page (https://guides.github.com/features/mastering-markdown/) was super helpful in showing how to format the code for embedding images